src/user/user.service.ts

```diff
@@ -0,0 +1,74 @@
+import { BadRequestException, Injectable } from '@nestjs/common';
+import { PrismaService } from 'src/libs/orm';
+// eslint-disable-next-line import/no-extraneous-dependencies
+import * as bcrypt from 'bcrypt';
+import { JwtService } from '@nestjs/jwt';
+import { Payload } from '../security/payload.interface';
+import { CreateUserDto, SignInDto } from './dto/create-user.dto';
+import { UpdateUserDto } from './dto/update-user.dto';
+
+@Injectable()
+export class UserService {
+  constructor(private prisma: PrismaService, private jwtService: JwtService) {}
+
+  async create(createUserDto: CreateUserDto) {
+    const { email, password, confirmPassword } = createUserDto;
+    const isExist = await this.prisma.user.findUnique({
```

Comment:
```suggestion
    const user = await this.prisma.user.findUnique({
```

boolean이 나오는 것이 아니라 User 타입이 나오는 것이기 때문에 isExist는 맞지 않는 변수명인 것 같습니다.

Reply:
수정했습니당